src/dynamixel/servos/protocol_specific_pacquets.hpp

```diff
@@ -0,0 +1,118 @@
+#ifndef DYNAMIXEL_SERVOS_PROTOSPECIFICPACQUETS_HPP_
```

Comment:
The name of the file shouldn't be `protocol_specific_packets`? And not `protocol_specific_pacquets`?